wonil/top_styles.css

```
@@ -0,0 +1,131 @@
+.top_top{
+    display: grid;
+    grid-template-columns: 950px 200px;
+    grid-template-rows: 30px 60px;
+    gap: 10px;
+}
+.cut{color: gray; opacity: 0.2;}
+.bar_1{
+    grid-column: 2/3;
+    grid-row:1/2;
+    display: flex;
+    font-size: 12px;
+    float: right;
+    margin: 20px;
+}
+.register{color: rgb(95, 0, 128);}
+.cm_center_parent_ul{
+    list-style-type: none; 
+    padding: 0; 
+    margin: 0;
+}
+.cm_center_ul li a{
+    background-color: white; 
+    text-decoration: none; 
+    display: block;
+}
+.cm_center_ul{
+    list-style-type: none;
+    display: none; 
+    position: absolute;
+}
+.cm_center_parent_li:hover > .cm_center_ul{display: block;}
+.bar_2{
+    grid-column: 1/3;
+    grid-row:2/3;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    top: 100px; left: 100px;
+}
+.top_top_left{display: flex; gap: 20px; margin: 0 0 0 20px;}
+.kurly_link{display: flex; gap: 10px; align-items: center; font-size: 15px;}
+.market_kulry{color: rgb(95, 0, 128);}
+.beauty_kulry{opacity: 0.2;}
+.beauty_kulry:hover{opacity: 1.0;}
+.top_top_middle{
+    display: flex; align-items: center;
+    background-color: white;
+    border-style: solid;
+    border-width: 2px;
+    border-color: rgb(95, 0, 128);
+    border-radius: 5px;
+    margin: 0 30px 0 0;
+}
+.text_bar{width: 350px; border-style: none; outline-style: none;}
+.search_btn{border-style: none; background-color: white;}
+.top_top_right{display: flex; gap: 10px; margin: 0 30px 0 0;}
+.location_btn:hover{content: url(./images/location_hover.png); width: 40px; height: 40%;}
+.like_btn:hover{content: url(./images/like_hover.png); width: 40px; height: 40%;}
+.cart_btn:hover{content: url(./images/cart_hover.png); width: 40px; height: 40%;}
+
+.top_bottom{
+    display: flex;
+    justify-content: space-between;
+}
+.top_bottom_left{
+    display: flex;
+    align-content: center;
+    height: 40px;
+}
+.category_btn{
+    display: flex; 
+    gap: 10px; 
+    align-items: center; 
+    margin: 0 0 0 20px; 
+    background-color: white;
+    border-style: hidden;
+}
+.category_parent_ul{
+    list-style-type: none; 
+    padding: 0; 
+    margin: 0;
+}
+.category_ul li a{
+    background-color: white; 
+    text-decoration: none; 
+    display: block;
+}
+.category_ul{
+    list-style-type: none;
+    display: none; 
+    position: absolute;
+}
+.category_parent_li:hover > .category_ul{display: block;}
+.top_bottom_middle{
+    width: 500px;
+    display: flex;
+    justify-content: space-between;
+    align-self: center;
+} 
+.list_1, .list_2, .list_3, .list_4{
+    font-size: medium;
+}
+.list_1:hover, .list_2:hover, .list_3:hover, .list_4:hover{
+    color: rgb(95, 0, 128);
+    text-decoration: underline;
+}
+.top_bottom_right{
+    display: flex;
+    align-items: center;
+    margin: 0 30px 0 0;
+}
+.information_btn{
+    width: 150px;;
+    height: 30px;
+    background-color: white;
+    border-style: solid;
+    border-color: gray;

```

```suggestion
    border: 1px solid gray;
```
로 하지 않은 이유가 있나요? border에 스타일링이 들어갔다는 거는 선의 굵기도 필요하니, 같이 써주는 게 좋을 것 같습니다. :)

---

border : 1px solid gray; 처럼 주는 방식을 까먹었었습니다.. ㅠ감사합니다